src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -837,6 +837,15 @@ public async Task EnableDatabases()
             await ToggleDisableDatabases(disable: false);
         }
 
+        [RavenAction("/admin/databases/restart", "POST", AuthorizationStatus.Operator)]
```

Comment:
I think that database admin should be able to restart owned database. You have 'CanAccessDatabaseAsync' method in the request executor

Reply:
This endpoint also needs proxy in 6.0 so we can execute that on other nodes. Please apply it there after this is merged.